corehq/apps/app_manager/xform.py

```diff
@@ -970,6 +980,93 @@ def already_has_meta(self):
 
         return meta_blocks
 
+    def _add_usercase_bind(self, usercase_path):
+        self.add_bind(
+            nodeset=usercase_path + 'case/@case_id',
+            calculate=SESSION_USERCASE_ID,
+        )
+
+    def add_usercase(self, form):
+        from corehq.apps.app_manager.util import split_path
+
+        usercase_path = 'usercase/'
+        actions = form.active_actions()
+        usercase_bound = False
+
+        if 'update_case' in actions and hasattr(actions['update_case'], 'update'):
+            usercase_updates = {k[5:]: v for k, v in actions['update_case'].update.items() if k.startswith('user:')}
```

Comment:
I'm assuming the `5:` is cutting off the 'user:' prefix. Small, but might be nice if we had a constant for `"user:"`. Then we could reduce spelling errors ever and in this scenario we could do `k[len(USERCASE_PREFIX):]` and it'd be clear that we're chopping off the user prefix